proposals/digest.md

```diff
@@ -0,0 +1,45 @@
+# OCI go-digest project proposal
+
+## Abstract
+opencontainers/image-spec#486 introduces a dependency on a stable upstream implementation of https://github.com/docker/go-digest, which was recently broken out of the https://github.com/docker/distribution project.
+
+This package has been instrumental in providing a strong hash-identity implementation in Go and I hope to extend this to OCI.
+
+Let's support this by moving this into a https://github.com/opencontainers/go-digest project specifically oriented towards providing this functionality throughout the container ecosystem. While this package does support opencontainers/image-spec, it is broadly useful in other image formats or outside image formats.
+
+Having a solid, battle-proven, common digest implementation in OCI for use in and outside the image-spec will ensure long lasting security and interoperability throughout the container ecosystem.
+
+## Proposal
+With repositories under the http://github.com/opencontainers organization:
+
+Rename (transfer) https://github.com/docker/go-digest would become https://github.com/opencontainers/go-digest.
+
+### Initial Maintainers
+Initial maintainers of the go-digest project would be seeded from the image-spec project:
+* Brandon Philips <brandon.philips@coreos.com> (@philips)
+* Brendan Burns <bburns@microsoft.com> (@brendandburns)
+* Jason Bouzane <jbouzane@google.com> (@jbouzane)
+* John Starks <jostarks@microsoft.com> (@jstarks)
+* Jonathan Boulle <jon.boulle@coreos.com> (@jonboulle)
+* Stephen Day <stephen.day@docker.com> (@stevvooe) [lead maintainer]
```

Comment:
sure, we can just list the set of maintainers. It's simpler for us to just carry over the list of image-spec maintainers to start